services/multi_runtime/main.py

```
@@ -966,6 +966,16 @@ async def respond(request: Request) -> RespondResponse:
     generated_text = pipeline_result.generated_text
     duration = pipeline_result.audio_duration_sec
     active_precision = str(daemon_params.get("precision", "auto"))
+    post_response_cleanup = "none"
+
+    if request_payload.release_after_response:
+        try:
+            await asyncio.to_thread(daemon.unload_all)
+            post_response_cleanup = "unload_all"
+            logger.info("Post-response cleanup: unload_all completed")
+        except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)